tests/actions/PolicyCategoryTest.ts

```diff
@@ -16,9 +17,10 @@ describe('actions/PolicyCategory', () => {
         });
     });
 
+    let mockFetch: MockFetch;
     beforeEach(() => {
-        // @ts-expect-error TODO: Remove this once TestHelper (https://github.com/Expensify/App/issues/25318) is migrated to TypeScript.
         global.fetch = TestHelper.getGlobalFetchMock();
```

Comment:
Do we still need this line?

Reply:
We just need to remove the comment, this `global.fetch = TestHelper.getGlobalFetchMock();` is necessary for the test.